example/timer/odp_timer_accuracy.c

```diff
@@ -767,11 +759,20 @@ int main(int argc, char *argv[])
 		void *addr;
 		uint64_t size = test_global.tot_timers * sizeof(test_log_t);
 
+		test_global.file = fopen(test_global.filename, "w");
+		if (test_global.file == NULL) {
+			printf("Failed to open output file: %s\n",
```

Comment:
The actual error could also be printed (e.g. `perror()` or `strerror()`).

Reply:
Fixed in V2.